tests/console/validate_lvm_with_raid1.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: RAID1 on LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "y2_module_consoletest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
+
+sub run {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    $config = get_test_data();
+    $config->{expected_num_devs} = $config->{num_disks};
+    _check_lvm_partitioning($config);
+    _check_raid1_partitioning($config);
+    _remove_raid_disk($config);
+    _reboot();
+    $self->wait_boot;
+    _check_raid_disks_after_reboot($config);
+    _restore_raid_disk($config);
+}
+
+sub _check_lvm_partitioning {
+    my ($config) = @_;
+    my $activelvm = script_output q[lvscan | awk '{print $2}' | sed s/\'//g];
+    assert_equals $config->{lvm}->{lvpath}, $activelvm, "lv name is not the same. \n" . script_output q[lvscan];
+    my $pvname = script_output q[pvscan -s | grep '/dev'];
+    assert_equals $config->{lvm}->{pvname}, $pvname, "pv name is not the same.";
+}
+
+sub _check_raid1_partitioning {
+    my ($config) = @_;
+    record_info("raid1 name", $config->{raid1}->{name});
+    assert_script_run 'mdadm --detail ' . $config->{lvm}->{pvname};    # should correspond to $config->{raid1}->{name}
+
+    assert_script_run 'grep \'md0 : active ' . $config->{raid1}->{level} . '\' /proc/mdstat';
+
+    my $active_devs = script_output("mdadm --detail " . $config->{raid1}->{name} . " |grep \"Active Devices\" |awk '{ print \$4 }'");
+    assert_equals($active_devs, $config->{expected_num_devs}, "Active devices are different");
+
+    for (keys %{$config->{disks}}) {
```

Comment:
sorry, I didn't see that it was in a loop and make you delete the disks, please, add them back and remove `num_disks: 4`